src/main/scala/play/modules/reactivemongo/MongoController.scala

```diff
@@ -124,23 +124,29 @@ trait MongoController {
 
   /** Gets a body parser that will save a file sent with multipart/form-data into the given GridFS store. */
   def gridFSBodyParser(gfs: GridFS[JSONSerializationPack.type])(implicit readFileReader: Reads[ReadFile[JSONSerializationPack.type, JsValue]], ec: ExecutionContext): BodyParser[MultipartFormData[Future[ReadFile[JSONSerializationPack.type, JsValue]]]] = {
```

Comment:
I'm not sure this is the easier change adding an extra parameter `maxSize: Int = BodyParsers.parse.DefaultMaxDiskLength`, and passing this `maxSize` as the last parameter of `multipartFormData` should be sufficient.